.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "go.testEnvVars": {
+        "COMPLEMENT_BASE_IMAGE": "complement-synapse"
+    }
+}
```

Comment:
not sure if we should be commiting this?

Reply:
Ideally not, as it will make Dendrite people sad.

Reply:
Good spot! I must have added it by accident while merging in `main`.